### ISSUE TITLE
inputType="email" not supported 

### ISSUE THREAD

**Issue by asonni** [djorg83/react-bootstrap-sweetalert#27]
Hi,

I've noticed that `inputType="email"` not working correct the input accept a none valid email if `required ={true}`.
Any suggestions how can I fix this?

Thanks

**Comment by asonni** [comment]
When I added

`<SweetAlert input inputType="email" validationMsg="Please enter your email address" ... />`

I got the validation message if I didn't enter anything, but if I enter a non valid email the SweetAlert accept the value.

I suggest to add new property something like this  `validationEmailMsg="You must provide a valid email"`

Thanks

**Comment by djorg83** [comment]
Im publishing version 4.4 with support for this.  I'm also exposing a new prop `validationRegex` if anyone needs to override the regex used.

``` js
<SweetAlert 
	 input
         inputType="email" 
         validationMsg="Must be a valid email!"
	 title="Enter your email!" 
	 onConfirm={this.hideAlert} 
> 
 	Email is required
 </SweetAlert>
```

**Comment by djorg83** [comment]
I just published 4.4.1 and changed the default `validationMsg` to `'Please enter a valid email!'` for `inputType` of `'email'`

**Comment by asonni** [comment]
It works thank you very much 👍
